README.md

```diff
@@ -10,4 +10,29 @@
 
 A website for analyzing commuting patterns and more.
 
+<p align="center"><img src="screenshot.png" /></p>
+
 Check out our [blog post](https://blog.conveyal.com/conveyal-commute-identifying-mode-shift-opportunities-2f8666a3ee90) for more details!
+
+## Installation
+
+Commute requires the following to run:
+
+* node.js version 6
+* The [yarn](https://yarnpkg.com/en/) package manager
+* A mongodb database
+* API Keys for the [ESRI geocoder](https://developers.arcgis.com/features/geocoding/)
+* Version v2.4.1-9-g3be6daa of [r5](https://github.com/conveyal/r5/)
```

Comment:
I.e. this link? https://github.com/conveyal/r5/tree/3be6daa139de55582ecdd730ed38bee3fa7954ef

Reply:
I'll send you more details on slack.